waffle_hub/temp_utils/image/io.py

```diff
@@ -21,6 +21,18 @@ def save_image(output_path: Union[str, Path], image: Mat, create_directory: bool
         with open(str(output_path), mode="w+b") as f:
             img_arr.tofile(f)
 
+def save_images(output_path: Union[str, Path], images: list[Mat], create_directory: bool = False) -> None:
```

Comment:
it seems better to change the function name. Functionally, it appears to merge multiple images into one and save them as a single image. However, based on the function name alone, it looks like it saves each image individually into separate directories.

Reply:
I would consider naming it `batch_save_images`. Thank you for participating in the review!

Reply:
I think..
I'm also confused about `batch_save_images` naming.
How about naming it `save_concat_image`?